app/lib/features/backups/dialogs/show_confirm_reset_identity.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:acter/common/toolkit/buttons/danger_action_button.dart';
+import 'package:acter/common/toolkit/buttons/primary_action_button.dart';
+import 'package:acter/features/backups/dialogs/show_recovery_key.dart';
+import 'package:acter/features/backups/providers/backup_manager_provider.dart';
+import 'package:acter/features/backups/providers/backup_state_providers.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:acter/l10n/generated/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final passwordFormKey = GlobalKey<FormFieldState>(
+  debugLabel: 'Password Form Field',
+);
+
+class _ShowConfirmResetIdentityDialog extends ConsumerStatefulWidget {
+  const _ShowConfirmResetIdentityDialog();
+
+  @override
+  ConsumerState<ConsumerStatefulWidget> createState() =>
+      __ShowConfirmResetIdentityDialogState();
+}
+
+class __ShowConfirmResetIdentityDialogState
+    extends ConsumerState<_ShowConfirmResetIdentityDialog> {
+  final formKey = GlobalKey<FormState>(debugLabel: 'Password Form');
+  bool showInput = false;
+  final passwordController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    final lang = L10n.of(context);
+    return Form(
+      key: formKey,
+      child: AlertDialog(
+        title: Text(lang.encryptionBackupResetIdentityKeyTitle),
+        content: Container(
+          constraints: const BoxConstraints(maxWidth: 500),
```

Comment:
If we are just using `constraints` in `Container` then how about using `ConstrainedBox` instead of `Container`?

```suggestion
        content: ConstrainedBox(
          constraints: const BoxConstraints(maxWidth: 500),
```

Reply:
was just copy pasted code